### PR TITLE
tested, reformatted, moved file into .../janhektor/...

### PR DESCRIPTION
**Pull request by Scriptim**
well, hope this is useful

description:
"This is a class that allows you to log all activities of the plugin. Every class should static import ~/Log.* - this could be really helpful to ease bugfixes. Server admins can upload the log file, that is created in the DataFolder so that devs can exactly see, where the fault occures. Use logInfo(String msg), logWarning(String msg) or logConfig(String msg) to log 'msg' with the level info/warning/config. Optionally you can use logXXX(String msg, true) to print the log message in the server console"

### CODE REVIEW

**Comment by Scriptim** [comment]
like i said, this should be used in every class to ease bugfixes

**Comment by Rincewind34** [comment]
You have to observe our conventions

**Comment by Rincewind34** [comment]
Sry for German: Dein Einrücken ist (sry für die Formulierung) totaler Bullshit, wenn nicht sogar "Nicht Vorhanden". Drück mal 'Strg+Shift+F' in eclipse

**Comment by Scriptim** [comment]
@Rincewind34 You want me to observe the conventions, but you don't even? This is an english project, so use english... Well, there're indents, they're not 'not-existent'. I just added a commit that's called "larger indents", so I hope u r stasified

File: src/de/janhektor/community/utils/Log.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	  date = new Date();
+	  dateTime = "[" + dateFormat.format(date) + "]";
+   }
+
+   /**
+	* log a message with level 'INFO'
+	*
+	* @param msg   the message to be logged
+	* @param print wether 'msg' schould be printed in console
+	* @throws IOException could not close BufferedWriter
```
Comment (Scriptim) [inline src/de/janhektor/community/utils/Log.java]: when there's an update these lines will be removed

**Comment by Janhektor** [comment]
Hi Scriptim!

Now I have decided to merge your changes into our master branch.
Thanks for you involvement.
I hope, you're still interested in these project.

Regards,
Janhektor

**Comment by Scriptim** [comment]
@Janhektor ok thx, I'd prescribe, that every class must use this

**Comment by Janhektor** [comment]
I know, but currently we aren't using it.
Maybe I'll publish one extra tutorial to integrate the log-system.

**Comment by Scriptim** [comment]
Hey @Janhektor I've some problems with pushing, so can u pls add a file for me? That's be great. It's a file for log conventions, content:



Classes should (not forced) use the log feature (de.janhektor.community.utils.Log)
 exceptions: abstract classes, interfaces, enums, annotations

 They should use these templates:
   logWarning(e.getLocalizedMessage());
      in every catch-block
   -logInfo("Executed command " + label + " by " + sender.getName());
      at the end of every onCommand()-Block (or similar)
   -logInfo("Triggered " + event.getEventName() + " for player " + event.getPlayer());
      in the first line of every EventHandler-block

  Other log messages must start with an uppercase letter

  Class should static import Log.*
    -> import static de.janhektor.community.utils.Log.*;

**Comment by Janhektor** [comment]
Hey, 

currently we don't use the log feature as I wrote. 
Additional, wildcard imports (like de.janhektor.community.utils.Log.*) are forbidden by convention.

**Pull request merged**